src/nvim/testdir/test_timers.vim

```diff
@@ -14,13 +15,17 @@ func MyHandlerWithLists(lists, timer)
   let x = string(a:lists)
 endfunc
 
+func s:assert_inrange(lower, upper, actual)
+  return assert_inrange(a:lower, LoadAdjust(a:upper), a:actual)
+endfunc
+
 func Test_oneshot()
   let g:val = 0
   let timer = timer_start(50, 'MyHandler')
   let slept = WaitFor('g:val == 1')
   call assert_equal(1, g:val)
   if has('reltime')
-    call assert_inrange(40, 120, slept)
+    call s:assert_inrange(40, 120, slept)
```

Comment:
would be clearer if LoadAdjust() was explicitly called here

Reply:
Yup, but you merged it already. :P I guess we just silently do it the next time that file is changed in any way.